site/content/docs/user/known-issues.md

```diff
@@ -17,20 +17,21 @@ It may additionally be helpful to:
 - reach out and ask for help in [#kind] on the [kubernetes slack]
 
 ## Contents
-* [Failures involving mismatched kubectl versions](#failures-involving-mismatched-kubectl-version)
+* [kubectl version skew](#kubectl-version-skew)
```

Comment:
should start with caps for uniformity `Kubect Version Skew` :/

Reply:
I've never seen kubectl capitalized, but your version is missing the `l` :stuck_out_tongue: 

`kubectl` is the correct name imo, it's a binary. similar to `systemd`

https://kubernetes.io/docs/reference/kubectl/overview/ has a singla capital kubectl for starting the sentence. when a name in a heading it is lower case unlike eg `Kubernetes`.

Reply:
with `kind` I've given up and generally use same case, with `KIND` or `KinD` used to help disambiguate (or `sigs.k8s.io/kind`) but for tools with better recognition I prefer to follow upstream..

Reply:
I just was doing a diff against the header it links and the others links underneath it ... just bikeshedding a bit before taking some 💤 
👋